core/lib/server/pub.db.test.ts

```diff
@@ -846,6 +846,36 @@ describe("getPubsWithRelatedValuesAndChildren", () => {
 			members: users.map((u) => ({ ...u, role: MemberRole.admin })),
 		});
 	});
+
+	it("should fetch a pub that has no pub values", async () => {
```

Comment:
a bug came up that our e2e tests kind of caught after this restructuring—creating a pub with no values was not "gettable" by `getPubsWithRelatedValuesAndChildren`. this adds a test to make sure we can still get it